.github/workflows/ci-test.yml

```diff
@@ -537,11 +533,14 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0 #get all the history!!!
       - uses: ./.github/actions/build-setup
-      - uses: actions/cache@v2.1.6
-        with:
-          path: "/opt/cargo/git\n/opt/cargo/registry\n/opt/cargo/.package-cache"
-          key: crates-${{ runner.os }}-${{ hashFiles('Cargo.lock') }}
-          restore-keys: "crates-${{ runner.os }}"
+      - uses: Swatinem/rust-cache@v1
+      - name: compile tests
+        run: |
+          rm -rf ./target/x-scratch # Work around bad cached data
```

Comment:
So this is due to not having a way to clear the cache...While testing I didn't realize that some codepaths created directories in `/target` resulting in subsequent jobs restoring the cache and using the bad data stored in these diem-specific stuff we store there. This temporarily works around the issue. in a number of weeks we can go and remove this. and maybe in the future there will be a button we can use to clear the cache.